tsconfig.json

```diff
@@ -2,6 +2,7 @@
   // extend OpenSearchDashboards's tsconfig, or use your own settings
   "extends": "../../tsconfig.json",
   "compilerOptions": {
+    "jsx": "react",
```

Comment:
It is a setting for using JavaScript and HTML syntax together to create React elements. The code already had this stylization but opening it in VS Code was showing 'bad lines' because without the main dashboard repo in the same directory as it was trying to parse the code as one or the other and could not understand the jsx/tsx elements being created. This setting is already in the Dashboards repo but not in our repo--here is the place in the main Dashboards repo where the setting is set:

<img width="816" alt="Screen Shot 2022-10-03 at 5 08 44 PM" src="https://user-images.githubusercontent.com/65832608/193682921-95090666-44fd-425a-9eb8-f183237d88fd.png">

Reply:
Nice, thanks for the detailed response.  Are there other settings that we should synchronize?  might be worthwhile to do in another PR

Reply:
The only other thing is the elastic/eui library but that is offered through the Dashboard integration when the plugin is installed.